docs/streams/developer-guide/config-streams.html

```diff
@@ -333,7 +334,6 @@ <h4><a class="toc-backref" href="#id23">num.standby.replicas</a><a class="header
               the <code>org.apache.kafka.streams.state.DslStoreSuppliers</code> interface.
             </td>
             <td><code>BuiltInDslStoreSuppliers.RocksDBDslStoreSuppliers</code></td>
-            <td>null</td>
```

Comment:
@Cerchie You added this line, and it seems to break the layout (this row has one more cell than all the other reviews). Did I fix it right?

Reply:
LGTM -- might have bee some merge conflict what was not resolved correctly, and I missed this during PR review.